### ISSUE TITLE
Visual Bug of Rampage Buff on 1h Axe "Vengeance"

### ISSUE THREAD

**Issue by Oorunee** [WarEmu/WarBugs#11001]
Not sure if this is of sufficient importance...
Playing as a Dwarf Slayer.

Both the use of "Vengeance" (1h axe) in appearance slot or equipped will cause the "Rampage" buff to be visually displaced in mid air adjacent to the axe instead of on the weapon itself as intended.

Steps to reproduce the problem:

Equip "Vengeance" or mog it in the appearance slot. Activate Rampage.

Makes me real sad, since my transmog game suffers from it! :P
Jokes/ would be awesome if you could resolve the issue though. 

Anyway thanks for all!
  
* Complete rules for the bugtracker https://www.returnofreckoning.com/forum/viewtopic.php?f=52&t=10366

**Comment by szejoza** [comment]
The same happens with most of 1h hammers, might be hard to find every glitch like that tho

**Comment by Oorunee** [comment]
I agree, just selfishly hoping that with time these visuals can be adressed one after the other... :)
no hurry anyway!

greetz
